src/InputField/README.md

```diff
@@ -45,13 +46,14 @@ Table below contains all types of the props available in InputField component.
 
 ### enum
 
-| type            | size        |
-| :-------------- | :---------- |
-| `"text"`        | `"small"`   |
-| `"number"`      | `"normal"`  |
-| `"email"`       |
-| `"password"`    |
-| `"passportid"`  |
+| type            | inputMode      | size        |
+| :-------------- | :------------- | :---------- |
+| `"numeric"`     | `"text"`       | `"small"`   |
+| `"tel"`         | `"number"`     | `"normal"`  |
+| `"decimal"`     | `"email"`      |
+| `"email"`       | `"password"`   |
+| `"url"`         | `"passportid"` |
```

Comment:
I don't see password and passportid anywhere else, typo?

Also don't we wanna support all possible modes? https://html.spec.whatwg.org/multipage/interaction.html#input-modalities:-the-inputmode-attribute